src/usercloudssdk/client.py

```diff
@@ -469,6 +469,23 @@ def CreateToken(
         j = self._post("/tokenizer/tokens", data=ucjson.dumps(body))
         return j["data"]
 
+    def LookupOrCreateTokens(
+        self,
+        data: list[str],
+        transformers: list[ResourceID],
+        access_policies: list[ResourceID],
+    ) -> list[str]:
+        body = {
+            "data": data,
+            "transformer_rids": [t.__dict__ for t in transformers],
+            "access_policy_rids": [a.__dict__ for a in access_policies],
+        }
+
+        j = self._post(
+            "/tokenizer/tokens/actions/lookuporcreate", data=ucjson.dumps(body)
```

Comment:
if you do `  "/tokenizer/tokens/actions/lookuporcreate", json=body)` the requests library will set the correct content type on the request. 
 the only issues is that it can't use a custom encoder....
oh well...